src/web/routes.rs

```diff
@@ -170,47 +211,42 @@ pub(super) fn build_axum_routes() -> AxumRouter {
             "/crate/:name/:version/source/*path",
             get_internal(super::source::source_browser_handler),
         )
+        .route(
+            "/-/rustdoc.static/*path",
+            get_internal(super::rustdoc::static_asset_handler),
+        )
+        .route(
+            "/-/storage-change-detection.html",
+            get_internal(|| async {
+                #[derive(Debug, Clone, serde::Serialize)]
+                struct StorageChangeDetection {}
+                crate::impl_axum_webpage!(
+                    StorageChangeDetection = "storage-change-detection.html",
+                    cache_policy = |_| CachePolicy::ForeverInCdnAndBrowser,
+                );
+                StorageChangeDetection {}
+            }),
+        )
+        .route_with_tsr(
+            "/crate/:name/:version/download",
+            get_internal(super::rustdoc::download_handler),
+        )
+        .route(
+            "/crate/:name/:version/target-redirect/*path",
+            get_internal(super::rustdoc::target_redirect_handler),
+        )
+        .route(
+            "/:crate/badge.svg",
```

Comment:
adding this temporary slightly changes behaviour. 

Axum doesn't have the `LegacySharedResourcesHandler` yet, 
so if a crate had a `badge.svg` in its static assets, theoratically that one would be served instead of the redirect. 

Since axum comes first now, we would always serve the redirect.